### PR TITLE
Babylon various ideas hidde

### PR DESCRIPTION
**Pull request by HiddeLekanne**
A few changes based on the Ideas I had written down the last few weeks.
 -- removal of random bandits as it is just toxic to the game
 -- shortening the customs union and state transfer diplomatic plays
 -- added a land trade capacity modifier which is based on average infrastructure

### CODE REVIEW

File: common/on_actions/babylon_infamy_rebalance_on_actions.txt [hunk 1]
```diff
@@ -1,8 +1,50 @@
...
+		}
+
+		set_local_variable = {
+			name = gained_land_trade_capcity
+			value = {
+				add = local_var:total_infrastructure
+				divide = local_var:number_of_states
+				subtract = 50 # counting from 50 infrasturucture
+				divide = 8 # every 8 infrastructure average is 1 land_trade capacity
+			}
```
Comment (Danarca) [inline common/on_actions/babylon_infamy_rebalance_on_actions.txt:30]: Wait this works!?

File: common/on_actions/babylon_infamy_rebalance_on_actions.txt [hunk 2]
```diff
@@ -1,8 +1,50 @@
...
+
+		if = {
+			limit = {
+				local_var:gained_land_trade_capcity >= 1
+			}
+
+			add_modifier = { 
+				name = modifier_land_trade_capacity
+				multiplier = local_var:gained_land_trade_capcity
+				months = 1
```
Comment (Danarca) [inline common/on_actions/babylon_infamy_rebalance_on_actions.txt]: It feels like the monthly_pulse comes at a random time each month, so a country could potentially skip a week or two of this modifier, and also stack it =/

**Review by Danarca — APPROVED** [review]
I assume the `escalation >= 60` bit is necessary to combat that bug you've mentioned?
Assuming that the monthly_pulse comes at the same time each month and thus doesn't need removal of the old modifier I approve this :)

File: common/on_actions/babylon_infamy_rebalance_on_actions.txt (hunk 1 above)
Comment (HiddeLekanne) [reply common/on_actions/babylon_infamy_rebalance_on_actions.txt:30]: yes yes

File: common/on_actions/babylon_infamy_rebalance_on_actions.txt (hunk 2 above)
Comment (HiddeLekanne) [reply common/on_actions/babylon_infamy_rebalance_on_actions.txt]: done

**Pull request merged**